include/unifex/linux/io_uring_context.hpp

```diff
@@ -439,7 +441,7 @@ class io_uring_context::read_sender {
         sqe.rw_flags = 0;
         sqe.user_data = reinterpret_cast<std::uintptr_t>(
             static_cast<completion_base*>(this));
-        sqe.__pad2[0] = sqe.__pad2[1] = sqe.__pad2[2] = 0;
+        std::fill(std::begin(sqe.__pad2), std::end(sqe.__pad2), 0);
```

Comment:
Under the new layout of the structure we would no longer be zeroing out the `buf_index`, `personality`, etc, fields.

Maybe this code should take a different approach and just zero out the entire `io_uring_sqe` struct in `try_submit_io()` and then have the lambdas populate only the fields that they actually need to set for that operation?